packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelForm.tsx

```diff
@@ -0,0 +1,392 @@
+              },
+            })
+            setIsShowingRestrictedValuesModal(false)
+          }}
+          onCancel={() => setIsShowingRestrictedValuesModal(false)}
+        />
+        <TextField
+          fullWidth
+          value={((columnModel as ColumnModelFormData)?.enumValues ?? []).join(
+            ', ',
+          )}
+          onClick={() => {
+            setIsShowingRestrictedValuesModal(true)
+          }}
+          disabled={canHaveRestrictedValues(
+            columnModel.columnType,
+            isJsonSubColumn,
+          )}
+          InputProps={{
+            // Is readOnly because edits are made with the JSONArrayEditorModal
+            readOnly: true,
+            sx: fieldSx,
+            inputProps: {
+              'aria-label': 'Restrict Values',
+            },
+          }}
+        />
+      </Box>
+      <Box>
+        <FormControl fullWidth>
+          <Select
+            label="Facet Type"
+            value={columnModel.facetType}
+            disabled={allowedFacetTypes === null}
+            onChange={e => {
+              dispatch({
+                type: 'setColumnModelValue',
+                columnModelIndex,
+                jsonSubColumnModelIndex: jsonSubColumnIndex,
+                value: {
+                  ...columnModel,
+                  facetType: e.target.value as FacetType,
+                },
+              })
+            }}
+            sx={fieldSx}
+          >
+            {(allowedFacetTypes ?? []).map((value, index) => {
+              return (
+                <MenuItem value={value} key={index}>
+                  {value === undefined ? '' : getFacetTypeFriendlyName(value)}
+                </MenuItem>
+              )
+            })}
+          </Select>
+        </FormControl>
+      </Box>
+      {isJsonSubColumn && (
+        <>
+          <Box>{HIERARCHY_VERTICAL_LINE_COMPONENT}</Box>
+          <Box></Box>
+          <Box
+            sx={{
+              gridColumn: '3 / span 7',
+            }}
+          >
+            <TextField
+              placeholder={'JSON Path'}
+              value={(columnModel as JsonSubColumnModelFormData).jsonPath}
+              onChange={e => {
+                dispatch({
+                  type: 'setColumnModelValue',
+                  columnModelIndex,
+                  jsonSubColumnModelIndex: jsonSubColumnIndex,
+                  value: {
+                    ...(columnModel as JsonSubColumnModelFormData),
+                    jsonPath: e.target.value,
+                  },
+                })
+              }}
+              fullWidth
+              InputProps={{
+                sx: fieldSx,
+                endAdornment: (
+                  <Tooltip
+                    title={
+                      <React.Fragment>
+                        <p>
+                          <Link
+                            href={
+                              'https://dev.mysql.com/doc/refman/8.0/en/json.html#json-path-syntax'
```

Comment:
While technical, this is probably the best documentation we can give without writing our own, since other implementations of JSON Path may offer features we don't support.